### ISSUE TITLE
Addon-Docs: source-loader has non-ES5-friendly template literals in output

### ISSUE THREAD

**Issue by adamdoyle** [storybookjs/storybook#7388]
The output file `@storybook/source-loader/dist/events.js` is below:

_(comment mine)_
```javascript
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.STORY_EVENT_ID = exports.ADDON_ID = void 0;
const ADDON_ID = 'storybook/source-loader';
exports.ADDON_ID = ADDON_ID;
const STORY_EVENT_ID = `${ADDON_ID}/set`;  // <-- template literal
exports.STORY_EVENT_ID = STORY_EVENT_ID;
```

Consequently, in IE11, the stories don't load and it logs this error:
```
SCRIPT1014: Invalid character
File: vendors~main.fedb7751a9c76d1fa4ef.bundle.js, Line: 201390, Column: 24
```

Line `201390:24` is the first backtick in `` `${ADDON_ID}/set` ``

**System:**
 - Browser: IE11
 - Framework: polymer
 - Version:  5.2.0-alpha.40

@libetl @shilman

**Comment by adamdoyle** [comment]
There's also some arrow functions, etc. throughout.

Haven't seen that with the other `@storybook`-scoped packages.

I notice that [tsconfig.json](https://github.com/storybookjs/storybook/blob/32d3d476c436134803071a3aec1a3cfcf01ff0b7/tsconfig.json#L15) sets its target to `es5` and that [.babelrc.js](https://github.com/storybookjs/storybook/blob/32d3d476c436134803071a3aec1a3cfcf01ff0b7/.babelrc.js#L94) sets an override for the (otherwise ES5) target for source-loader (and a few others) to be `node: '8.11'`.  According to [node.green](https://node.green/), version `8.11` supports template literals, so it would make sense that preset-env would leave them in.

I suspect that the general strategy in the `.babelrc.js` is to target browser-exposed code against ES5 and target node-only code against node-v8.11.

What I don't know is whether `@storybook/source-loader` was intentionally added as a Node target (because maybe[?] it needs to be ES6 for some reason) or whether it was just an oversight.

**Comment by libetl** [comment]
Well source-loader is almost nodejs only. We need to rethink the way storysource uses the source-loader to completely get rid of it in the frontend

**Comment by libetl** [comment]
Unfortunately it seems that this task is more urgent than expected. Michael ? @shilman

**Comment by shilman** [comment]
@libetl I think migrating Storysource to use parameters & removing decorators from source-loader is important from a code hygiene standpoint. I'm hoping this issue can get solved in the process.

**Comment by libetl** [comment]
makes sense

**Comment by libetl** [comment]
Maybe I will find some time during the week end to help.

**Comment by shilman** [comment]
That would be 💯💯💯

**Comment by shilman** [comment]
Fixed in 5.2.0-beta.5?
